_layouts/layout.html

```diff
@@ -82,6 +82,12 @@
     </script>
     {% endif %}
 
+      
+
+  <!-- Used for LATEX equations -->
+<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+<script type="text/javascript" src="/js/jq_mathjax_parse.js"></script>
```

Comment:
That's not a surprise! You need to explicitly enable MathJax support as
documented here:
https://github.com/RcppCore/rcpp-gallery/wiki/Contributing-to-the-Rcpp-Gallery#features

On Wed, Jun 1, 2016 at 4:17 PM, Dirk Eddelbuettel <notifications@github.com>
wrote:

> In _layouts/layout.html
> <https://github.com/RcppCore/rcpp-gallery/pull/86#discussion_r65434215>:
>
> > @@ -82,6 +82,12 @@
> >      </script>
> >      {% endif %}
> >
> > +
> > +
> > +  <!-- Used for LATEX equations -->
> > +<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
> > +<script type="text/javascript" src="/js/jq_mathjax_parse.js"></script>
>
> Are you sure we need this? We already have working mathjax support, see eg this
> post <http://gallery.rcpp.org/articles/EM-algorithm-example/>.
>
> @jjallaire <https://github.com/jjallaire> Any comments?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RcppCore/rcpp-gallery/pull/86/files/ab94898f290f6f43701b2ee3c2f229fd2e968ea9#r65434215>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGXx-waECn3CB1Fxk18wRcXgeo98RtKks5qHejtgaJpZM4Ir1B3>
> .
>